Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Building the image
```

Comment:
We may not need this Dockerfile, leaving in for now.
It is a part of the default gauge-taiko project template.